### ISSUE TITLE
Seems that destName is now required

### ISSUE THREAD

**Issue by cjh1** [girder/girder#2463]
I have the following parameters description:

```python
  ...
  .modelParam('id', 'The ID of the calculation.', model='calculation',
                    plugin='molecules', level=AccessType.ADMIN))
  def update_properties(self, calculation, params):
```
This no longer works with master, I have to add destName?

```python
   ...
  .modelParam('id', 'The ID of the calculation.', destName='calculation', model='calculation',
                    plugin='molecules', level=AccessType.ADMIN))
   def update_properties(self, calculation, params):
```

**Comment by zachmullen** [comment]
Sounds like a bug. If you're converting it, the modern way to do it is just to pass the class itself as the `model` parameter, then you can omit `destName` and `plugin`.

**Comment by cjh1** [comment]
Yes, it seems the ModelImporter.model(...) uses the module name (which is how it was done in the past), now the code uses model.name, which in my case they are different!

**Comment by cjh1** [comment]
I have a few plugins like this so would like to correct it in girder if possible.

**Comment by cjh1** [comment]
Seem like this is the offending [line](https://github.com/girder/girder/blob/master/girder/api/describe.py#L662), in the past model.name would have been the module name of the model?

**Comment by cjh1** [comment]
I think rather than model.name is should me info['model'] if destName is not provided?

**Comment by cjh1** [comment]
I think this is what we want? #2464

**Comment by cjh1** [comment]
Fixed by #2464
